src/universal/modules/meeting/components/MeetingAgendaCards/MeetingAgendaCards.js

```diff
@@ -32,44 +34,45 @@ const handleAddProjectFactory = (teamMemberId, agendaId) => () => {
 
 let s = {};
 
-const makeCards = (array, dispatch) => {
+const makeCards = (array, dispatch, myTeamMemberId) => {
   return array.map((outcome) => {
-    const key = `${outcome.type}OutcomeCard${outcome.id}`;
+    const {type, id, teamMember: {preferredName}, content, teamMemberId} = outcome;
+    const key = `${type}OutcomeCard${id}`;
+    const username = preferredName && preferredName.replace(/\s+/g, '');
     return (
       <div className={s.item} key={key}>
-        <AgendaCard form={key} dispatch={dispatch} outcome={outcome}/>
+        {(!content && myTeamMemberId !== teamMemberId) ?
```

Comment:
Same logic as the team dash, the guy who creates the outcome gets an autofocused card. Everybody else sees a NullCard until a value gets passed through